lib/translation.rb

```diff
@@ -45,15 +49,29 @@ def configure(&block)
         # include is private until Ruby 2.1
         Proxy.send(:include, GetText)
 
-        Proxy.bindtextdomain(TEXT_DOMAIN, {
-          :path           => @config.locales_path,
-          :output_charset => @config.charset
-        })
+        if @config.multi_domain
+          # TODO: Here we iterate over the langauges to initialize them
+          @config.domain_names.each do |domain|
+            Proxy.bindtextdomain(domain, {
+              path: @config.locales_path,
+              output_charset: @config.charset
+              })
+          end
+        else
+          Proxy.bindtextdomain(@config.text_domain, {
+            :path           => @config.locales_path,
+            :output_charset => @config.charset
+          })
+        end
```

Comment:
If `@config.multi_domain` false or not set, fall-through to default text_domain (app)